### PR TITLE
Update index.php

### LINKED ISSUE #4: Bring back Lil Pump!
Bring back Lil Pump!

### LINKED ISSUE #5: Update index.php
As a @marissuhhhh, I want to see my name on the Internet

### PR DESCRIPTION
**Pull request by marissuhhhh**
Fixes #4 and #5

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -225,6 +225,11 @@ function gtag(){dataLayer.push(arguments);}
 <td>Henna</td>
 <td>#2020</td>
 </tr>
+<tr>
+<td>Marissa</td>
+<td>MONDAY, FEBRUARY 10TH 2020</td>
```
Comment (drakeapps) [inline index.php]: Umm, I don't think this is the proper date formatting we use

**Review by drakeapps — CHANGES_REQUESTED** [review]

**Comment by drakeapps** [comment]
Also, could you change the music back to lil pump?

File: index.php (hunk 1 above)
Comment (jsbolton) [reply index.php]: I believe it's TWENTYMEMETEEN, year of our Lord

Comment (thornjad) [inline index.php]: It looks like you're trying to write a date. Would you like some help?

```suggestion
<td>January Trumpteeth, Twentymemeteen</td>
```

**Review by thornjad — CHANGES_REQUESTED** [review]

File: index.php [hunk 2]
```diff
@@ -225,6 +225,11 @@ function gtag(){dataLayer.push(arguments);}
...
+</tr>
+<tr>
```
Comment (thornjad) [inline index.php:239]: This is the worst indentation I've ever seen

File: index.php (hunk 1 above)
Comment (marissuhhhh) [reply index.php]: y'all suck

**Review by thornjad — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (marissuhhhh) [reply index.php]: what is this

File: index.php (hunk 2 above)
Comment (marissuhhhh) [reply index.php:239]: omg 🙄

File: index.php [hunk 3]
```diff
@@ -225,6 +225,11 @@ function gtag(){dataLayer.push(arguments);}
...
 </table>
 </div>
 <!-- Looking at the source?  Great!  Have a damn good day -->
```
Comment (thornjad) [inline index.php]: Did you look at the source? Did you have a damn good day?

**Review by thornjad — CHANGES_REQUESTED** [review]

**Review by thornjad — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (thornjad) [reply index.php]: That's the date

File: index.php (hunk 2 above)
Comment (thornjad) [reply index.php:239]: THIS IS THE WORST INDENTATION

**Review by thornjad — APPROVED** [review]

**Review by thornjad — CHANGES_REQUESTED** [review]
Where is lil pump?

**Review by thornjad — CHANGES_REQUESTED** [review]
Where is lil pump?

**Pull request merged**